feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/README.md

```diff
@@ -41,14 +44,16 @@ For prefixes:
 
 Parameters:
 
-*   graceful-restart/config/enabled
-*   graceful-restart/config/restart-time
-*   graceful-restart/config/stale-routes-time
-*   graceful-restart/config/helper-only
+*   /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/graceful-restart/config/enabled
+*   /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/graceful-restart/config/helper-only
+*   /network-instances/network-instance/protocols/protocol/bgp/global/graceful-restart/config/restart-time
+*   /network-instances/network-instance/protocols/protocol/bgp/global/graceful-restart/config/stale-routes-time
 
 ## Telemetry Parameter Coverage
 
-*   afi-safis/afi-safi/afi-safi-name
-*   afi-safis/afi-safi/graceful-restart/state/advertised
-*   afi-safis/afi-safi/graceful-restart/state/peer-restart-time
-*   afi-safis/afi-safi/graceful-restart/state/received
+*   /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/afi-safi-name
+*   /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/graceful-restart/state/advertised
+*   /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/graceful-restart/state/peer-restart-time
+*   /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/graceful-restart/state/received
+*   /network-instances/network-instance/protocols/protocol/bgp/global/graceful-restart/state/restart-time
+*   /network-instances/network-instance/protocols/protocol/bgp/global/graceful-restart/state/stale-routes-time
```

Comment:
Add section for ## gNOI RPC coverage
* gNOI.BGP:ClearBGPNeighborRequest
* gNOI.BGP:ClearBGPNeighborResponse

Reply:
Please check my last comment.

Reply:
If the device supports [RFC8538](https://www.rfc-editor.org/rfc/rfc8538.html) then one can use the [BGP reset soft option](https://www.rfc-editor.org/rfc/rfc8538.html#section-4.1) which is also defined in [gnoi.ClearBGPNeighborRequest with mode=SOFT](https://github.com/openconfig/gnoi/blob/main/bgp/bgp.proto#L41) .

If you feel we need to keep a [RFC 4724](https://www.rfc-editor.org/rfc/rfc4724) specific test, that is ok.  RFC8538 compliant behavior, using gnoi.ClearBGPNeighborRequest with mode=SOFT  could be an additional subtest.